pkg/domain/infra/abi/network.go

```diff
@@ -87,7 +87,16 @@ func (ic *ContainerEngine) NetworkRm(ctx context.Context, namesOrIds []string, o
 					// Without the force option, we return an error
 					return reports, errors.Errorf("%q has associated containers with it. Use -f to forcibly delete containers", name)
```

Comment:
Can you add "and pods" to the end of the error here ("to forcibly delete containers and pods")